### ISSUE TITLE
About the FPS 

### ISSUE THREAD

**Issue by wjq777777** [HZAI-ZJNU/Mamba-YOLO#37]
Hi, Author,

I would like to know roughly how the FPS metric of the model you proposed is defined. You compared it with most of the YOLO series detection models, but the biggest feature of this series of models is real-time performance. Your paper does not reflect real-time performance.

**Comment by EthanW-coder** [comment]
Indeed, the models and papers that have been published our are preliminary results and we will update this metric in future public papers.
